CONTRIBUTING.md

```diff
@@ -586,33 +586,6 @@ or that are not particularly interesting.
 Here are a few bits and pieces that are referenced from some of the
 scenarios in this guide.
 
-### Pull Request Guidelines
```

Comment:
I think it is still important to reference the pull request guidelines.
Linking is probably ok.